evm-contracts/test/v0.6/ValidatingAggregator.test.ts

```diff
@@ -0,0 +1,946 @@
+import {
+  contract,
+  helpers as h,
+  matchers,
+  setup,
+} from '@chainlink/test-helpers'
+import { assert } from 'chai'
+import { ethers } from 'ethers'
+import { ValidatingAggregatorFactory } from '../../ethers/v0.6/ValidatingAggregatorFactory'
+import { AnswerValidatorTestHelperFactory } from '../../ethers/v0.6/AnswerValidatorTestHelperFactory'
+import { GasGuzzlerFactory } from '../../ethers/v0.6/GasGuzzlerFactory'
+import { HistoricDeviationValidatorFactory } from '../../ethers/v0.6/HistoricDeviationValidatorFactory'
+import { FlagsFactory } from '../../ethers/v0.6/FlagsFactory'
+import { SimpleWriteAccessControllerFactory } from '../../ethers/v0.6/SimpleWriteAccessControllerFactory'
+
+let personas: setup.Personas
+const provider = setup.provider()
+const linkTokenFactory = new contract.LinkTokenFactory()
+const fluxAggregatorFactory = new ValidatingAggregatorFactory()
+const answerValidatorFactory = new AnswerValidatorTestHelperFactory()
+const validatorFactory = new HistoricDeviationValidatorFactory()
+const flagsFactory = new FlagsFactory()
+const acFactory = new SimpleWriteAccessControllerFactory()
+const gasGuzzlerFactory = new GasGuzzlerFactory()
+const emptyAddress = '0x0000000000000000000000000000000000000000'
+
+beforeAll(async () => {
+  personas = await setup.users(provider).then(x => x.personas)
+})
+
+describe('ValidatingAggregator', () => {
```

Comment:
At a glance, it looks like a lot of these tests are copy/pasted from the FluxAggregator.test.ts file. Should we just test the changes that ValidatingAggregator introduces?

Reply:
I've consolidated this all into Flux Aggregator now.